mautic/mautic
# decision's negative path actions aren't triggering as expected

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | possibly https://github.com/mautic/mautic/pull/14089, https://github.com/mautic/mautic/pull/14089



#### Description:


Negative path action should be execute immediately just after define waiting time of decision if positive path does not executed till defined time.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a campaign
3. Create send email action
4. Create decision and add 1 action in positive and 1 action in negative path
5. In negative path action, define configuration wait time ex. 5 min and add values in between the hours.
6. If action is on negative path of decision then we should ignore waiting time on execution of action.

#### Other areas of Mautic that may be affected by the change:
1. Campaign Conditions
2.
